app/code/Magento/PageCache/Model/System/Config/Backend/AccessList.php

```diff
@@ -24,7 +24,7 @@ public function beforeSave()
         parent::beforeSave();
 
         $value = $this->getValue();
-        if (!is_string($value) || !preg_match('/^[\w\s\.\-\,\:]+$/', $value)) {
+        if (!is_string($value) || !preg_match('/^[\w\s\.\-\,\:(\/\d+)?]+$/', $value)) {
```

Comment:
1. The pattern places `(\/\d+)?` inside a character class `([...])`, which means these are treated as individual characters rather than a sequence.
2. The pattern remains overly permissive and would validate strings like "this is: a more / or less, valid string.... oh, lets add numbers 12345" which aren't valid IP addresses or CIDR notations